src/thor/worker.cc

```diff
@@ -96,6 +96,16 @@ thor_worker_t::thor_worker_t(const boost::property_tree::ptree& config,
 thor_worker_t::~thor_worker_t() {
 }
 
+std::string serialize_to_pbf(Api& request) {
+  std::string buf;
+  if (!request.SerializeToString(&buf)) {
+    LOG_ERROR("Failed serializing to pbf in Thor::Worker - trace_route");
+    throw valhalla_exception_t{401, boost::optional<std::string>(
```

Comment:
Is `valhalla_exception_t` appropriate here instead of `std::runtime_error`?

Reply:
Looking at other exceptions thrown here, seems like `valhalla_exception_t` is the right choice.

Reply:
Aye, removed the `runtime` error for `valhalla_exception_t`